grizli/prep.py

```diff
@@ -3700,9 +3855,14 @@ def tweak_align(direct_group={}, grism_group={}, max_dist=1., n_min=10, key=' ',
         return True
 
     # Redrizzle
-    bits = 576
-    driz_cr_snr = '8.0 5.0'
-    driz_cr_scale = '2.5 0.7'
+    if isJWST:
+        bits = 1
```

Comment:
Again, the default values above are for WFC3/IR, and I don't think they should need to be that different for JWST.  But who knows for now, we'll have to adjust based on the real data.

Reply:
Fixed: 09c29642f78b0417e5251f73af0c9e8fc47b53a1